### PR TITLE
Contract

### PR DESCRIPTION
**Pull request by OuChiaYun**
檔案在 test.sol
試寫，然後應該沒有任何的防呆(警告)

er.png 供參，但他挺錯的，有時間我會畫一張對的

遍歷計算all_money 系列的code我會再補上

### CODE REVIEW

File: packages/contracts/package-lock.json
Comment (NOOBDY) [inline packages/contracts/package-lock.json:1]: this shouldn't be here

File: packages/contracts/package.json
Comment (NOOBDY) [inline packages/contracts/package.json:1]: does this affect anything?

**Review by NOOBDY — CHANGES_REQUESTED** [review]
is there any testing function i can use or is there any way i could test it?

File: packages/contracts/package-lock.json
Comment (OuChiaYun) [reply packages/contracts/package-lock.json:1]: delete

File: packages/contracts/package.json
Comment (OuChiaYun) [reply packages/contracts/package.json:1]: no,
我只是以為他跟lock一樣是多出來的

**Review by NOOBDY — APPROVED** [review]

**Pull request merged**